### DISCUSSION TITLE
Problem with watertight geometry workflow definition

### DISCUSSION THREAD

**Discussion by hugoboudier** [ansys/pyfluent#2224] [General]
Hello. I am a new PyFluent user and have been trying to run the sample notebook at [this link](https://fluent.docs.pyansys.com/version/stable/examples/00-fluent/external_compressible_flow.html#sphx-glr-examples-00-fluent-external-compressible-flow-py). I have run the imports and the meshing setup but the workflow definition is giving me this error I don't understand. After research online, I have not been able to find the fix. 

Any help would be appreciated! 
Note: I have installed ansys-fluent-core, ansys-fluent-parametric and ansys-fluent-visualization

**Comment by raph-luc** [comment]
This message you shared is probably the reason:
![image](https://github.com/ansys/pyfluent/assets/132297401/b129aa6e-6d34-4083-b2ff-927c0cd3c1bf)

How did you install PyFluent? Which Fluent version are you using?

**Comment by hugoboudier** [comment]
I am using fluent 22R2. I created a venv and then pip install ansys-fluent-core. If I add the show_gui bool set to "true" in the launch_fluent command above a fluent window does appear

**Reply by raph-luc** [reply]
It seems that there is something wrong with your installation, I tested starting from a clean environment and the steps up to and including "Import CAD and set length units" work fine on 22R2.

Try starting from a clean environment, then `pip install ansys-fluent-core==0.18.1`. Alternatively, try the development install described here: https://fluent.docs.pyansys.com/version/stable/getting_started/installation.html#install-the-package

However, in general, the examples are tested for Fluent 2023 R2. I can confirm Fluent 2022 R2 does not have the new APIs required to run through this entire example, it will stop working at the `local_sizing.AddChildAndUpdate()` command (that requires at least Fluent 2023 R1).
